### DISCUSSION TITLE
A question of DeepSurvK

### DISCUSSION THREAD

**Discussion by JasonLnzi** [arturomoncadatorres/deepsurvk#201] [General]
Hello Arturo, 
Thank you for your work concerning DeepSurvK. I read the example of "00 - Understanding DeepSurv (using Keras)" and notice that the code in "Define the loss function" of "num_observed_events = tf.math.cumsum(E)" results in a set of numbers rather than a number. This causes the iterations to fail and error of "Invalid loss, terminating training". The right code should be "num_observed_events = tf.reduce_sum(E_train)" and it will work fine.

**Comment by TanweiY** [comment]
Hello Arturo,

many thanks for developing this module. But I'm wondering if it is possible to change the batch size to 32/16, rather than the whole set of patient. Because my model is huge. However, when I tried to do so, it always report error, something related to the self-defined Loss function. It seems that the dimensionality of E and the self-defined batch size (e.g., 16) is not compatible.
